service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcDocumentTranslator.java

```diff
@@ -24,28 +26,40 @@
     public String translateToMhsOutboundRequestData(
         DocumentTaskDefinition taskDefinition, String response) {
 
-        var binary = fhirParseService.parseResource(response, Binary.class);
```

Comment:
The whole change in this file is not correct.
This `translateToMhsOutboundRequestData` is only used here https://github.com/nhsconnect/integration-adaptor-gp2gp/blob/23435fc5ea5b0abd930eb2143945df563f630c9b/service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskExecutor.java#L40 so it always operate on FHIR Bundle

I think this file shouldn't have any change. It has nothing to do with large content